nntrainer/compiler/tflite_interpreter.cpp

```diff
@@ -12,14 +12,21 @@
 #include <tflite_interpreter.h>
 
 #include <fstream>
+#include <memory>
 #include <string>
 
 #include <tf_schema_generated.h>
 
 #include <nntrainer_error.h>
+#include <node_exporter.h>
+#include <tensor.h>
+
+#define UNUSED(x) x __attribute__((unused))
+#define WILL(x) ;
 
 static constexpr const char *FUNC_TAG = "[TFLITE INTERPRETER] ";
 
+namespace nntrainer {
```

Comment:
is it intended? ```namespace nntrainer{ namespace{ ```??

Reply:
Yes, those functions are not used outside of this file.